azure-pipelines.yml

```diff
@@ -0,0 +1,89 @@
+trigger:
+  batch: true
+  branches:
+    exclude:
+      - gh-pages
+  paths:
+    exclude:
+      - docs/*
+      - "**/*.md"
+      - .travis.yml
+
+stages:
+  - stage: Build
+    displayName: 🚧
+    jobs:
+      - job: BuildOnWindows
+        displayName: "Build with VS 2019 on Windows"
+        pool:
+          vmImage: windows-2019
+        steps:
+          - bash: dotnet help
+            displayName: "Initialize .NET Core Environment"
+          - bash: dotnet build --configuration Release
+            displayName: "Build Solution"
+          - publish: .
+            artifact: windows-build
+          - bash: dotnet build --configuration Release --version-suffix "ci-${BUILD_BUILDID}+${BUILD_SOURCEVERSION}" src/Telegram.Bot/
+            displayName: "Build Package with CI version"
+          - task: NuGetCommand@2
+            displayName: "Publish CI Package to Azure Artifacts"
+            inputs:
+              command: push
+              publishVstsFeed: Telegram.Bot
+              packagesToPush: src/**/*-ci-*.nupkg
```

Comment:
Do we really need to publish nuget packages from each branch to the azure feed? And the order of packages will be completely meaningless.

Reply:
Well, we can publish the artifacts for `develop` branch only and that provides sequential package versions.

Publishing a package for every single build enables the users to try out the latest features that are not stable, approved, or tested yet. We probably cannot guarantee the sequential order of the package versions but instead expect the users to be more involved in the build process and know what package version they are using.

Reply:
I'm not sure if this is needed outside of develop branch. Though I don't really mind it.